project4/index.html

```diff
@@ -0,0 +1,257 @@
+                is
+                the
+                school he started with Leslie Jensen-Inman to create industry-ready User Experience Designers. UIE
+                is
+                Center
+                Centre’s professional development arm, dedicated to understanding what it takes for organizations to
+                produce
+                competitively great products and services.
+
+                In the 39 years he's been in the tech field, he's worked with hundreds of organizations, written two
+                books,
+                published hundreds of articles and podcasts, and tours the world speaking to audiences everywhere.
+                When
+                he
+                can, he does his laundry in Andover, Massachusetts.</p>
+        </div>
+
+        <div class="jake">
+            <img class="jake-photo" src="images/jake.jpg" alt="picture of jake knapp">
+            <p class="speakers-name">jake knapp</p>
+            <p class="speakers-introduction">Jake Knapp is the author of Make Time and the New York Times
+                bestseller,
+                Sprint.
+
+                Jake spent 10 years at Google and Google Ventures, where he created the Design Sprint. He has since
+                coached
+                teams like Slack, Uber, 23andMe, LEGO, and The New York Times on the method.
+
+                Previously, Jake helped build products like Gmail, Google Hangouts, and Microsoft Encarta. He is
+                currently
+                among
+                the world’s tallest designers.</p>
+        </div>
+
+        <div class="kate">
+            <img class="kate-photo" src="images/kate.jpg" alt="picture of kate retter">
+            <p class="speakers-name">kate retter</p>
+            <p class="speakers-introdeuction">Kate is an entrepreneur+designer and Principal at Intelleto, where she
+                creates
+                visual
+                explanations that make complex ideas simple, memorable and shareable. Kate pioneered the UX learning
+                track
+                at
+                Tradecraft, Co-founded Luxr.co, and was Senior Practitioner at UX consultancy Adaptive Path. She
+                co-hosts
+                the
+                NSFW podcast What Is Wrong With UX with Laura Klein, and blogs at intelleto.com.</p>
+        </div>
+    </section>
+
+
+    <section class="tickets" id="tickets">
+        <h2>Tickets</h2>
+        <div class="tickets-content">
+            <p class="tickets-question">Ready to Join Us?</p>
+            <p class="tickets-introduction">2-day conference jam-packed with workshops, talks, andexperiences for
+                designers
+                and
+                product people.</p>
+            <p class="tickets-money">2-Day Pass: <span>$899</span></p>
+            <p class="tickets-money">Sunday workshop: <span>$599</span></p>
+        </div>
+        <a class="button" href="http://">Buy Tickets</a>
```

Comment:
If you're including a blank link in your HTML, add it as a hashtag/pound sign, like so:
```suggestion
        <a class="button" href="#">Buy Tickets</a>
```